dll/win32/shell32/res/rgs/copyaspathmenu.rgs

```diff
@@ -12,15 +12,12 @@ HKCR
     }
     NoRemove AllFilesystemObjects
     {
-        NoRemove shell
+        NoRemove shellex
         {
-            ForceRemove copyaspath
+            NoRemove ContextMenuHandlers
             {
-                val MUIVerb = s '@%MODULE%,-30328'
-                val Extended = s ''
-                DropTarget
+                ForceRemove CopyAsPathMenu = s '{F3D06E7C-1E45-4A26-847E-F9FCDEE59BE0}'
                 {
-                    val CLSID = s '{F3D06E7C-1E45-4A26-847E-F9FCDEE59BE0}'
```

Comment:
Does this mean it always gets loaded, even when it's not used?

Reply:
It does mean that some COM and menu related code always runs when you open the context menu, yes.

Since it is already inside shell32 it's not as bad as a LoadLibrary on something else but  Microsoft encouraged everyone to move to static entries for a reason.

Reply:
I don't think that this feature considered a bloat (also MoveTo and CopyTo menu) because they are built in to shell32 and the Copy as Path menu code only does a few thing (return from `QueryContextMenu` method) when it's not used.

Reply:
Imho everything that is not used by the user (and is always loaded) can be considered as being bloat. Personally I never used the MoveTo/CopyTo/CopyAsPath ever for 18 years. So for me it is bloat. For others which use all of the 3 it is not. Therefore I do think that features which extend the shell are best handled in a plugin manner.

Sure, considering a few strings and functions as bloat, is a more strict point of view, regarding to OOBE, IE-shell-integration, updater-services and the like. Still trying to keep the shell sleek is a very legit concern. Bloat is by definition something that creeps into software slowly and steadily. It always starts with just a few wasted bytes here and there.

Reply:
So, which is the best design choice for this feature?
- Compile-time switch
- Separate DLL
- None of these ...

Reply:
I do think a separate dll which can be unregistered by `regsvr /u` is the best way to go. Like zipfldr does for example.
it could be turned on by default even. That wouldn't really hurt anybody.

Reply:
@JoachimHenze Thanks for the info. I think this is a good idea too.

Reply:
> I do think a separate dll which can be unregistered by `regsvr32 /u` is the best way to go. Like zipfldr does for example. it could be turned on by default even. That wouldn't really hurt anybody.

Now that would add a ton more overhead than the current implementation in master.

Reply:
Even if the the library-approach would imply a small overhead in itself, it at least gives the power to the end-user, about what he wants, and what he doesn't want to have. I do think that is important especially for features that are beyond our target.

Reply:
@JoachimHenze 
> I do think that is important especially for features that are beyond our target. And the price has to be payed only by those who actually want to use it.

And then, we will have to rewrite the added features later to remove these 
unnecessarily added overheads when we changed the target Windows version.

I think making the shell light and adding customization support is a good idea but that's beyound the scope of this PR. I will keep the design as is (CopyTo+MoveTo menu in shell32 and Windows 10-like CopyAsPath menu).

Reply:
Sure, I didn't request introducing the modularity within the scope of this PR ofc.